project/BuildDependencies/DownloadBuildDeps.bat

```diff
@@ -29,14 +37,26 @@ md %TMP_PATH%
 cd scripts
 
 SET FORMED_OK_FLAG=%TMP_PATH%\got-all-formed-packages
-REM Trick to preserve console title
+:: Trick to preserve console title
```

Comment:
They're equivalent, as far as cmd.exe is concerned. But I think it reads more naturally to a human if the characters that introduce a comment are not themselves an English word. I can switch them all to "rem" if someone feels strongly about it.